llvm-spirv/lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1815,6 +1815,54 @@ LLVMToSPIRVBase::transValueWithoutDecoration(Value *V, SPIRVBasicBlock *BB,
       BM->addExtension(ExtensionID::SPV_INTEL_hw_thread_queries);
     }
 
+    // TODO: it's W/A for intel/llvm to prevent not fixed SPIR-V consumers
```

Comment:
I suggest we open an issue and put a link here. I hope the issues will be scanned more often than SPIR-V writer source code.

Reply:
I've opened https://github.com/intel/llvm/issues/7592 keeping in mind having 2 customizations to add (this one and one from https://github.com/intel/llvm/pull/7669 ). I'll update the description once we have these PRs merged.

Reply:
I don't mind having one issue.